src/hotspot/cpu/aarch64/assembler_aarch64.cpp

```diff
@@ -124,7 +119,7 @@ extern "C" {
       Disassembler::decode((address)start, (address)start + len);
   }
 
-  JNIEXPORT void das1(uintptr_t insn) {
+  void das1(uintptr_t insn) {
```

Comment:
Doesn't this need to be exported for debugging on Windows-Aarch64?

Reply:
Ah, right, there is a Windows/aarch64 port. Forgot about it. :(

I don't know who maintains/cares about that port anymore. Pinging @mo-beck @lewurm @luhenry. Do anyone of you care about this anymore? If not, do you know if anyone has taken up the baton? (Or maybe it's time to retire the Windows/aarch64 port)

Reply:
cc @karianna @gdams, is there a Windows/AArch64 owner?

> Or maybe it's time to retire the Windows/aarch64 port

I guess Windows/AArch64 as a platform hasn't really taken off _yet_, give it a few more years 😉 I certainly would not advocate for removing it.

Reply:
Personally I have never used `das`/`das1` on any AArch64 port when debugging.  I guess it was somewhat handy when the backend was originally developed.  I would argue that the built-in disassembler of any debugger and `disnm` from `debug.cpp` are good enough, and thus `das`/`das1` can be removed.  What do you think @theRealAph ?

@

Reply:
Hi all - yes the Windows Arm port is maintained by Microsoft's JDK team. How can we help in this particular case?

Reply:
I assume you want to know if we use DAS/DAS1 for debugging purposes on that platform?

Reply:
I use `das` every day. In contrast, I don't know what `das1` is for.

Reply:
It does not sound like anyone object to the removal of `JNIEXPORT` for `das1`, then.

Or should I just remove the entire function, if it serves no purpose that any current maintainers know or care about?

Reply:
You can probably remove the entire function given it's an indirection to `das` anyway. I remember using it for debugging during the first stages of the Windows-AArch64 port but I've long forgotten about it.

Reply:
@luhenry @magicus das1 was added when we were implementing the AArch64 port in order to help us integrate the debugger support we provided when running JITted code on our AArch64 simulator into gdb. Now that we have real hardware it is redundant.

Reply:
Thanks @adinn for the confirmation! I've now removed it.

Reply:
Removal makes sense - thank you @lewurm, @luhenry, and @adinn for your input!